docs/en/stack/security/securing-communications/tutorial-tls-certificates.asciidoc

```diff
@@ -0,0 +1,127 @@
+[role="xpack"]
+[testenv="trial"]
+[[encrypting-communications-certificates]]
+=== Generate certificates
+
+In a secured cluster, {es} nodes use certificates to identify themselves when
+communicating with other nodes. 
+
+The cluster must validate the authenticity of these certificates. The 
+recommended approach is to trust a specific certificate authority (CA). Thus 
+when nodes are added to your cluster they just need to use a certificate signed 
+by the same CA. 
+
+. Use the `elasticsearch-certutil` command to generate a CA and certificates and
+private keys for each node in your cluster. 
++
+--
+You can let the tool prompt you for information about each node in your cluster,
+or you can supply that information in an input file. For example, create a
+`test-cluster.yml` file in one of your {es} nodes:
+
+[source,yaml]
+----
+instances:
+  - name: "node-1" <1>
+    dns: 
+      - "localhost"
+    ip:
+      - "127.0.0.1"
```

Comment:
I think it might be worthwhile to also include the loopback IPv6 address, `::1` under IP to avoid anyone having issues around that.

Reply:
Thanks, Jay.  I added those values.